tools/init2json.py

```diff
@@ -27,19 +27,26 @@ def parse_channel_from_config(config, id):
     return channel
 
 
-def ini2json(f, output='collections'):
+def ini2json(f, output='collections', metadata='metadata'):
+    filename, _ext = os.path.splitext(os.path.basename(f))
+    metadata_file = os.path.join(metadata, f'{filename}.json')
+    output_path = os.path.join(output, f'{filename}.json')
+
     config = configparser.ConfigParser()
     config.read(f)
 
-    out = {'channels': []}
+    out = {'channels': [], 'metadata': {}}
 
     channel_ids = config['kolibri']['install_channels']
-    for id in channel_ids.split():
+    channels = channel_ids.split()
+    for id in channels:
         channel = parse_channel_from_config(config, id)
         out['channels'].append(channel)
 
-    filename, _ext = os.path.splitext(os.path.basename(f))
-    output_path = os.path.join(output, f'{filename}.json')
+    with open(metadata_file) as f:
+        out['metadata'] = json.load(f)
+        out['metadata']['channels'] = len(channels)
```

Comment:
It is not very useful to pre-calculate the lenght of channels here. The explore plugin has the array of channels already so it can count them.